src/coreclr/src/vm/jitinterface.cpp

```diff
@@ -99,6 +99,35 @@ GARY_IMPL(VMHELPDEF, hlpDynamicFuncTable, DYNAMIC_CORINFO_HELP_COUNT);
 
 #else // DACCESS_COMPILE
 
+long g_cbILJitted = 0;
```

Comment:
long in C++ (sometimes 32-bit and sometime 64-bit) is not same as long in C# (always 64-bit). It is best to use explicit size types in C++, e.g. `int64_t`.